WeakAuras/RegionTypes/DynamicGroup.lua

```diff
@@ -809,15 +824,11 @@ local function modify(parent, region, data)
     end
   end
 
-  function region:DoPositionChildren()
-    WeakAuras.StartProfileSystem("dynamicgroup")
-    WeakAuras.StartProfileAura(data.id)
-    local newPositions = {}
-    self.growFunc(newPositions, self.sortedChildren)
-    for index, regionData in ipairs(self.sortedChildren) do
+  function region:positionSortedChildren(pos, children)
```

Comment:
nit:
```suggestion
  function region:PositionSortedChildren(pos, children)
```
Also I somewhat expected this to just be extracted as a local helper function, and not to a method. But it's not something i particularly care about.

Reply:
i did it like that to keep access to self